test/giovanni-averaging-service/environment.yaml

```
@@ -0,0 +1,13 @@
+name: papermill-giovanni-averaging-service
+channels:
+  - conda-forge
+  - nodefaults
+dependencies:
+  - python=3.11exit
```

I think this is a typo, I can't build an environment but will assume you mean for this to be 3.11?
```suggestion
  - python=3.11
```